examples/typescript-stack/lib/cdk-typescript-stack.ts

```diff
@@ -95,8 +20,10 @@ export class CdkTypeScriptStack extends Stack {
       enableDatadogASM: true,
       flushMetricsToLogs: true,
       site: "datadoghq.com",
-    });
+    };
+
+    const datadogLambda = new Datadog(this, "Datadog", datadogProps);
```

Comment:
Just double checking, the change from `DatadogLambda` to `Datadog` was intended in this PR?

Reply:
Good catch. Yes it's intended. A PR for renaming the interfaces was reverted, so I want to change the example stack back to use all the old interfaces. Let me also rename the variable `datadogLambda` to `datadog`.

Reply:
Cool, that sounds good